apsuite/commissioning_scripts/calc_orbcorr_mat.py

```diff
@@ -20,8 +20,8 @@ def __init__(self, model, acc, dim='4d'):
             raise Exception('Set models: BO or SI')
         self.dim = dim
         self.bpms = self._get_idx(self.fam_data['BPM']['index'])
-        self.ch = self._get_idx(self.fam_data['CH']['index'])
-        self.cv = self._get_idx(self.fam_data['CV']['index'])
+        self.chidx = self._get_idx(self.fam_data['CH']['index'])
```

Comment:
I think these types (`chidx`, `cvidx`) will be frequently recurring objects across our libraries. I think we should carefully choose names and use them consistently, if possible. Our libraries will be more intuitive this way. This applies to many categories (`ch`, `psname`, `bpm`, `element`, etc). I propose `ch_idx`, for example, as word separation makes names clearer and it is being used in other repos.

also, `bpm` in this code should follow the same name logic as `ch` and `cv`. I usually reserve names like `bpm`, `pwrsupply`, `pvs` to objects of the corresponding types, not associated indices. I think standards like this make code easier to read, and interfaces more intuitive to use. what do you think?